zerver/fixtures/bugdown-data.json

```diff
@@ -421,6 +421,18 @@
       "bugdown_matches_marked": true
     },
     {
+      "name": "timestamp",
+      "input": "!time(Jun 5th 2017, 10:30PM)",
+      "expected_output": "<p><span class=\"timestamp\" value=\"1496701800\"><img alt=\":clock4:\" class=\"emoji\" src=\"/static/generated/emoji/images/emoji/unicode/1f553.png\" title=\":clock4:\"><span value=\"1496701800\">Jun 5th 2017, 10:30PM</span></span></p>",
+      "bugdown_matches_marked": false
+    },
+    {
+      "name": "timestamp_incorrect",
+      "input": "!time(hello world)",
+      "expected_output": "<p><span class=\"timestamp\"><img alt=\":clock4:\" class=\"emoji\" src=\"/static/generated/emoji/images/emoji/unicode/1f553.png\" title=\":clock4:\"><span>hello world</span></span></p>",
+      "bugdown_matches_marked": false
+    },
```

Comment:
Well, these are the tests for the markdown processor; at least the specific examples we wrote tests for should match, right?

(Also, we should have both a unix timestamp and full date test cases, right?)